std/math.d

```diff
@@ -7384,6 +7649,20 @@ do
     }
 }
 
+/// ditto
+Unqual!(CommonType!(T1, T2)) poly(T1, T2, int N)(T1 x, ref const T2[N] A) @safe pure nothrow @nogc
+if (isFloatingPoint!T1 && isFloatingPoint!T2 && N > 0 && N <= 10)
+{
+    // statically unrolled version for up to 10 coefficients
```

Comment:
Why up to 10? Is that a certain sweet spot between compile time and runtime speed? Or just arbitrary.

Reply:
Just an arbitrary guess wrt. sweet spot for code-size/inline-ability. I searched std.math for P and Q arrays, and the longest one had 9 coefficients. ;)